integration-tests/devtools/src/test/java/io/quarkus/platform/catalog/CatalogProcessorTest.java

```diff
@@ -29,6 +30,7 @@ void testCategoryOrder() {
     }
 
     @Test
+    @Disabled("See https://github.com/quarkusio/quarkus/pull/24201#issuecomment-1063800991")
```

Comment:
Are you sure it's still failing because I fixed it yesterday and it passed locally?

https://github.com/quarkusio/quarkus/commit/29aa7daccc2032dd3ddb894e9a1b87ffe476e698

Reply:
Ah ok, my PR must have been in between.. Thanks @gsmet :)

Still this test needs to be improved, I will do that..